benchmarks/decoders/benchmark_readme_data.json

```diff
@@ -1,225 +1,201 @@
-        "video": "/tmp/torchcodec_benchmarking_videos/nasa_960x540_206s_30fps_yuv420p.mp4"
-    },
-    {
-        "decoder": "torchvision[video_reader]",
-        "description": "100 next()",
-        "fps_median": 220.82061460318502,
-        "fps_p25": 222.48418761498934,
-        "fps_p75": 218.84938919419412,
-        "frame_count": 100,
-        "iqr": 0.007465103641152382,
-        "median": 0.452856270596385,
-        "type": "100 next()",
+        "decoder": "torchcodec[cuda]",
+        "description": "dataloader 10 decode_and_transform()",
+        "fps_median": 14.26538824481199,
+        "fps_p25": 14.26538824481199,
+        "fps_p75": 14.26538824481199,
+        "frame_count": 640,
+        "iqr": 0.0,
+        "median": 44.86383328773081,
+        "type": "batch 10 x decode_and_transform()",
         "video": "/tmp/torchcodec_benchmarking_videos/nasa_960x540_206s_30fps_yuv420p.mp4"
     },
     {
-        "decoder": "torchaudio",
+        "decoder": "torchcodec[cuda]",
         "description": "uniform 10 seek()+next()",
-        "fps_median": 10.54381002725746,
-        "fps_p25": 10.612585301638902,
-        "fps_p75": 10.516430888261935,
+        "fps_median": 3.3734269771663272,
+        "fps_p25": 3.3734269771663272,
+        "fps_p75": 3.3734269771663272,
         "frame_count": 10,
-        "iqr": 0.008615483529865742,
-        "median": 0.9484237646684051,
+        "iqr": 0.0,
+        "median": 2.964344587177038,
         "type": "uniform seek()+next()",
         "video": "/tmp/torchcodec_benchmarking_videos/nasa_960x540_206s_30fps_yuv420p.mp4"
     },
     {
-        "decoder": "torchaudio",
+        "decoder": "torchcodec[cuda]",
         "description": "random 10 seek()+next()",
-        "fps_median": 7.206591092692971,
-        "fps_p25": 7.231744857114745,
-        "fps_p75": 7.073061841995869,
+        "fps_median": 4.211141234921976,
+        "fps_p25": 4.211141234921976,
+        "fps_p75": 4.211141234921976,
         "frame_count": 10,
-        "iqr": 0.0310227214358747,
-        "median": 1.3876186218112707,
+        "iqr": 0.0,
+        "median": 2.374653197824955,
         "type": "random seek()+next()",
         "video": "/tmp/torchcodec_benchmarking_videos/nasa_960x540_206s_30fps_yuv420p.mp4"
     },
     {
-        "decoder": "torchaudio",
+        "decoder": "torchcodec[cuda]",
         "description": "100 next()",
-        "fps_median": 204.94896843488522,
-        "fps_p25": 206.320878222405,
-        "fps_p75": 203.29397516227598,
+        "fps_median": 107.24389854402447,
+        "fps_p25": 107.24389854402447,
+        "fps_p75": 107.24389854402447,
         "frame_count": 100,
-        "iqr": 0.007216569967567921,
-        "median": 0.4879263397306204,
+        "iqr": 0.0,
+        "median": 0.9324539797380567,
         "type": "100 next()",
         "video": "/tmp/torchcodec_benchmarking_videos/nasa_960x540_206s_30fps_yuv420p.mp4"
     },
     {
-        "cpu_count": 56,
+        "cpu_count": 22,
```

Comment:
In `generate_readme_data.py`, we should add something like:
```
  "is_cuda_available": str(torch.cuda.is_available()),
```
In the collection of the system stats.